playbooks/containerd-build/run.yaml

```diff
@@ -0,0 +1,29 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: config-golang
+      arch: arm64
+  tasks:
+    - name: Build containerd
+      shell:
+        cmd: |
+          set -xe
+          apt-get update
+          apt-get install -y btrfs-tools libseccomp-dev git pkg-config
+          echo $PATH
+          echo $GOPATH
+
+          if [ ! -d "$GOPATH/src/github.com/containerd" ] ; then
```

Comment:
if these two jobs are only run in `periodic` pipelines, we don't need the forks for this project.

Reply:
The forks project(theopenlab/containerd|envoy) is the demo to show how to trigger the job in pr.  : )